mysql/content.md

```diff
@@ -176,6 +176,17 @@ If the application you're trying to connect to MySQL does not handle MySQL downt
 
 If you start your `%%IMAGE%%` container instance with a data directory that already contains a database (specifically, a `mysql` subdirectory), the `$MYSQL_ROOT_PASSWORD` variable should be omitted from the run command line; it will in any case be ignored, and the pre-existing database will not be changed in any way.
 
+## Running as an arbitrary user
+
+If you know the permissions of your directory are already set appropriately (such as running against an existing database, as described above) or you have need of running `mysqld` with a specific UID/GID, it is possible to invoke this image with `--user` set to any value in order to achieve the desired access/configuration:
```

Comment:
We should mention that it needs to be any user other than `root` since root will step down to the mysql user of the image.

Reply:
Good point, updated!